.pre-commit-config.yaml

```diff
@@ -16,16 +16,16 @@ repos:
         exclude: .ipynb|.po|.xml
       - id: debug-statements
   - repo: https://github.com/psf/black
-    rev: 22.12.0
+    rev: 23.1.0
     hooks:
       - id: black
   - repo: https://github.com/PyCQA/isort
-    rev: 5.11.4
+    rev: 5.12.0
     hooks:
       - id: isort
         args: [ '--profile=black' ]
   - repo: https://github.com/nbQA-dev/nbQA
-    rev: 1.6.0
+    rev: 1.6.1
     hooks:
       - id: nbqa-black
         args: [ '--target-version=py37' ]
```

Comment:
Don't forget to update the `nbqa` black and isort dependency versions as well (unfortunately, `pre-commit autoupdate` doesn't do that for us yet).

Reply:
I don't want the latest versions, I want to fix the CI !